test/server/config_validation/cluster_manager_test.cc

```diff
@@ -0,0 +1,58 @@
+#include "envoy/json/json_object.h"
```

Comment:
Yeah -- once we've replaced the example config tests with this, adding more example configs sounds like a good way of upping the coverage. But we might as well get there first.